tests/sv/light_HassTurnOn.yaml

```diff
@@ -21,6 +21,7 @@ tests:
       slots:
         area: Kök
         domain: light
+    response: "Slog på"
```

Comment:
I'm not sure if "Slog på" is a correct response to turning on an areas lights. 
I think it would be:
`Påslaget` Eller `Tänt`

Reply:
I agree. But since the default response for the light domain seemed to be "Slog på [...]", i just went with that. I wanted to keep my first PR small. If i change the response in the test the test will fail, so i decided to keep the default response.

Reply:
Ok, I think it just to add
```
      lights: "Tände {{ slots.name }}"
```
Or:
```
      lights: "Tände lampan"
```
To https://github.com/home-assistant/intents/blob/main/responses/sv/HassTurnOn.yaml
 
Since `lights_area` is already defined.
And the same for HassTurnOff.yaml

Don't think it will make the PR to big for this case.